### PR TITLE
Added manager for posting, updating and displaying entries.

### PR DESCRIPTION
**Pull request by artpard**
Finally! I am not proud from some pieces of the code here, but for now we do not have time to make it more readable and efficient. I think summer will be good time to refactor a major part of the code.

### CODE REVIEW

File: app/src/main/java/org/cleancity/cleancity/CleanCityRestClient.java [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
+import org.apache.http.Header;
+
 
 public class CleanCityRestClient {
-        private static final String BASE_URL = "http://10.0.2.2:8000/";
+    private static final String BASE_URL = "http://150.254.78.139/";
+
+    private static AsyncHttpClient client = new AsyncHttpClient();
+
+    public static AsyncHttpClient getClient() {
```
Comment (adasikow) [inline app/src/main/java/org/cleancity/cleancity/CleanCityRestClient.java]: Unnecessary method

File: app/src/main/java/org/cleancity/cleancity/CleanCityRestClient.java [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
+        return client;
+    }
+
+    public static void setClient(AsyncHttpClient client) {
```
Comment (adasikow) [inline app/src/main/java/org/cleancity/cleancity/CleanCityRestClient.java]: This one is unnecessary too I guess

**Comment by adasikow** [comment]
Looks fine (except of those two methods) :+1:

**Comment by artpard** [comment]
All comments addressed. @adasikow please take a look again.

**Comment by adasikow** [comment]
:+1:

**Pull request merged**